modules/hcp-migrate-aws-multiarch-nodepools.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-manage/hcp-manage-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-migrate-aws-multiarch-nodepools_{context}"]
+= Creating node pools on the multi-architecture hosted cluster
+
+After transitioning your hosted cluster from single-architecture to multi-architecture, you must create node pools manually. You can create node pools on compute machines based on 64-bit AMD and 64-bit ARM architectures.
+
+.Procedure
+
+. Create node pools based on 64-bit ARM architecture by entering the following command:
++
+[source,terminal]
+----
+$ hcp create nodepool aws \
+--cluster-name <hosted_cluster_name> \// <1>
+--name <nodepool_name> \// <2>
+--node-count=<node_count> \// <3>
+--arch arm64
+----
+<1> Replace `<hosted_cluster_name>` with your hosted cluster name.
+<2> Replace `<nodepool_name>` with your node pool name.
+<3> Replace `<node_count>` with integer for your node count, for example, `2`.
+
+. Create node pools based on 64-bit AMD architecture by entering the following command:
++
+[source,terminal]
+----
+$ hcp create nodepool aws \
+--cluster-name <hosted_cluster_name> \// <1>
+--name <nodepool_name> \//
+--node-count=<node_count> \//
+--arch amd64
+----
+
+.Verification
+
+* Verify that the multi-architecture {product-title} release image is updated for your node pools by entering the following command:
```

Comment:
Should this be something like `Verify the node pool is using the multi-architecture release image by entering the following command`? It kind of reads like a multi-arch release image was updated from some previous version.

Reply:
True. Updated the instruction.